configs/stage/roles/insights.json

```diff
@@ -12,6 +12,20 @@
           "permission": "advisor:*:*"
         }
       ]
+    },
+    {
+      "name": "Advisor Viewer",
+      "display_name": "RHEL Advisor viewer",
+      "description": "Be able to read Advisor data.",
+      "system": true,
+      "platform_default": false,
+      "admin_default": false,
+      "version": 11,
```

Comment:
please start with version 1

Reply:
my bad.. I thought need to follow the other version on same file.. :)